SparkPost/2web2ui
# SA-1498: use metrics rollup as query param

### What Changed
 - Removed metrics rollup header and added query param

### How To Test
 - Add `use-metrics-rollup` ui option and check that rollup=true is in the api query.
